docs/database/gravity-groups.md

```diff
@@ -0,0 +1,35 @@
+## Group management
+Any domain on the individual lists as well as the adlists entries can be managed through groups. This allows not only grouping them together to highlight their relationship, but also enabling/disabling them together if one, for instance, wants to visit a specific service only temporarily.
+
+Groups are defined in the `group` table and can have an optional description in addition to a mandatory name of the group.
+
+Label | Type | Uniqueness enforced | Content
+----- | ---- | ------------------- | --------
+`id` | integer | Yes | Unique ID for database operations
+`enabled` | boolean | No | Flag whether domains in this group should be used<br>(`0` = disabled, `1` = enabled)
+`name` | text | No | Mandatory group name
+`comment` | text | No | Optional field for arbitrary user comments
+
+Group management is implemented using so called linking tables. Hence, it is possible to
+- manage domains in no, one, or multiple groups at the same time,
```

Comment:
"manage domains in no" sounds a little odd, perhaps instead "associate domains with any number of groups".